imod_coupler/drivers/ribametamod/exchange.py

```diff
@@ -194,7 +199,7 @@ def to_modflow(self, realised: NDArray[np.float64]) -> None:
             # in which case the Modflow demand was POSITIVE, otherwise the correction is 0
             self.mf6_active_river_api_packages[key].rhs[:] = -(
                 np.maximum(self.demands_mf6[key], 0.0)
-                * (1 - self.mapping.mod2rib[key].transpose().dot(realised_fraction))
+                * (1 - self.mapping.map_mod2rib[key].transpose().dot(realised_fraction))
```

Comment:
I think it's better to do this transpose once, then just re-use it.
Generally a transpose results in new allocations etc.